cmd/controlplane/management_controller.go

```diff
@@ -31,10 +31,11 @@ func newManagementControllerCommand() *cobra.Command {
 
 			version := versionpkg.GetVersion()
 
-			log.WithFields(log.Fields{
-				"version": version.Version,
-				"commit":  version.GitCommit,
-			}).Info("Starting Kargo Management Controller")
+			log := logging.LoggerFromContext(ctx)
```

Comment:
See [this comment](https://github.com/akuity/kargo/pull/1791/files#r1556679011) -- same concern.